### PR TITLE
tests: crypt_util: encrypt error condition

### PR DESCRIPTION
**Pull request by dachary**
Signed-off-by: Loic Dachary <loic@dachary.org>

### CODE REVIEW

File: securedrop/tests/test_crypto_util.py [hunk 1]
```diff
@@ -23,3 +24,23 @@ def test_clean(self):
             crypto_util.clean('foo bar`') # backtick is not currently allowed
         with self.assertRaises(crypto_util.CryptoException):
             crypto_util.clean('bar baz~') # tilde is not currently allowed
+
+    def test_encrypt(self):
+        journalist, _ = utils.db_helper.init_journalist()
+        source, _ = utils.db_helper.init_source()
+        replies = utils.db_helper.reply(journalist, source, 1)
```
Comment (fowlslegs) [inline securedrop/tests/test_crypto_util.py]: Why are you creating a `Reply` here? Think this should probably just be removed.

File: securedrop/tests/test_crypto_util.py [hunk 2]
```diff
@@ -23,3 +24,23 @@ def test_clean(self):
             crypto_util.clean('foo bar`') # backtick is not currently allowed
         with self.assertRaises(crypto_util.CryptoException):
             crypto_util.clean('bar baz~') # tilde is not currently allowed
+
+    def test_encrypt(self):
+        journalist, _ = utils.db_helper.init_journalist()
```
Comment (redshiftzero) [inline securedrop/tests/test_crypto_util.py]: You can remove this `init_journalist()` line as you should only need 

```
source, _ = utils.db_helper.init_source()
```

for the setup for the function under test here

**Comment by dachary** [comment]
* split in two functions
* removed the journalist & reply lines

**Comment by dachary** [comment]
* used assertGreater instead of assert

**Comment by dachary** [comment]
@redshiftzero @fowlslegs hopefully addressed your comments, thanks for the careful review.

**Pull request merged**